README.md

```diff
@@ -1 +1,63 @@
-# todolist-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My ToDo List App
+
+> Description of Awesome Books.
+This is a todo list I built to store and list tasks with just Html Css and Javascript making use of webpack. <br>
+No frameworks Used
+
+## Description
+This app is created to manage tasks, Create and delete
+
+
+
+## Built With
+
+Html and Css and Js 
+
+## Library 
+[WebPack](https://webpack.js.org)
+
+
+## Getting Started
+
```

Comment:
- [ ] OPTIONAL
Kindly include under the _Getting Started_ section of your ReadMe, instructions on how to **setup** and **run** your application